examples/basic/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+// This configuration only applies to the package manager root.
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  ignorePatterns: ["apps/**", "packages/**"],
+  extends: ["@repo/eslint-config/library.js"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: true,
+  },
+};
```

Comment:
Should we adopt Flat Config from Eslint? https://eslint.org/docs/latest/use/configure/configuration-files-new

Reply:
I'm going to look into this as a follow-up!